lib/board.rb

```diff
@@ -0,0 +1,58 @@
+class Board 
+  attr_accessor :cells 
+
+ def initialize 
+   @cells = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
```

Comment:
You already have this code in your class; avoid repetition by calling the `reset!` method here.